extensions/antenna.sigmf-ext.md

```diff
@@ -35,3 +39,16 @@ the `annotations` object:
 |`azimuth_angle`|false|float|degrees|Angle of main beam in azimuthal plane from North.|
 |`elevation_angle`|false|float|degrees|Angle of main beam in elevation plane from horizontal.|
 |`polarization`|false|float|string|E.g. `"vertical"`, `"horizontal"`, `"slant-45"`, `"left-hand circular"`, `"right-hand circular"`.|
+
+# SigMF Collections
+
+The following fields are specificed in SigMF Collections.
+
+## Collection
+
+|name|required|type|unit|description|
+|----|--------------|-------|-------|-----------|
+
+|`azimuth_angle`|false|float|degrees|Angle of main beam in azimuthal plane from North.|
+|`elevation_angle`|false|float|degrees|Angle of main beam in elevation plane from horizontal.|
+|`polarization`|false|float|string|E.g. `"vertical"`, `"horizontal"`, `"slant-45"`, `"left-hand circular"`, `"right-hand circular"`.|
```

Comment:
not sure how i feel about `polarization` (a string) being included right now...probably needs to wait until we define how time-changing metadata values can be stored.